schemas/answers/text_field.json

```diff
@@ -22,6 +22,9 @@
       "suggestions_url": {
         "$ref": "https://eq.ons.gov.uk/common_definitions.json#/non_empty_string"
       },
+      "allow_multiple": {
+        "type": "boolean"
+      },
```

Comment:
Could `allow_multiple` be conditional on the existence of `suggestions_url`?

Reply:
We should consider nesting `url` and `allow_multiple` under a `suggestions` key, then `url` can be mandatory if `suggestions` exist:

```json
"suggestions": {
  "url": "",
  "allow_multiple": ""
}
```

Reply:
It would make the schema less ambiguous, so I think it would be beneficial to update.

Reply:
I've added nested url in suggestions and dependency of suggestion on url, need to fix runner as schemas and resolve script are already fixed and seems like working with new validator.